deebotozmo/cli.py

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env python3
+"""Cli module."""
+import asyncio
+import base64
+import configparser
+import itertools
+import json
+import logging
+import os
+import platform
+import sys
+import time
+from dataclasses import asdict
+from functools import wraps
+from typing import Any, Callable, Tuple
+
+import aiohttp
+import click
+
+from deebotozmo.commands import (
+    Charge,
+    CleanCustomArea,
+    CleanPause,
+    CleanResume,
+    CleanSpotArea,
+    CleanStart,
+    PlaySound,
+    SetFanSpeed,
+    SetWaterLevel,
+)
+from deebotozmo.ecovacs_api import EcovacsAPI
+from deebotozmo.ecovacs_mqtt import EcovacsMqtt
+from deebotozmo.events import (
+    BatteryEvent,
+    CleanLogEvent,
+    FanSpeedEvent,
+    MapEvent,
+    RoomsEvent,
+    StatsEvent,
+    WaterInfoEvent,
+)
+from deebotozmo.util import md5
+from deebotozmo.vacuum_bot import VacuumBot
+
+
```

Comment:
```suggestion
CONFIG_SECTION="GLOBAL"

```
You can use also another section name